components/automate-chef-io/content/docs/infra-server.md

```diff
@@ -156,6 +156,40 @@ Installations require elevated privileges, so run the commands as the superuser
 
 1. [Set up `knife`]({{< relref "infra-server.md#use-knife-with-chef-infra-server" >}}) for use with Chef Infra Server.
 
+## How to Add Chef Infra Server to a Chef Automate Installation
```

Comment:
May want to review the usage of the words "How to" in the level-two titles here. I would think it should be used for all of them or for none of them.